app/views/backend/purchases/show.html.haml

```diff
@@ -41,7 +41,7 @@
 
   - if resource.affair
     - c.cobble :affair do
-      = affair_of(resource, default: 'PurchasePayment')
+      = affair_of(resource, default: 'PurchasePayment', purchase_payment_attributes: { responsible_id: resource.responsible_id, paid_at: resource.payment_at, to_bank_at: resource.payment_at})
```

Comment:
Why the option name is `<model>_attributes` and not `params` or `url_options` or something more "URLesque"

Reply:
Yes, sorry, I forgot at that time that we can have other types... The most explicit name as parameter is: `new_<model>_url_options`. But your last proposal seems more sustainable.
Do you agree?

Reply:
I agree. I will implement the last proposal.